UI/GameInfoCache.cpp

```diff
@@ -598,14 +598,12 @@ class GameInfoWorkItem : public Task {
 					info_->ParseParamSFO();
 
 					if (info_->wantFlags & GAMEINFO_WANTBG) {
-						ReadFileToString(&umd, "/PSP_GAME/PIC0.PNG", &info_->pic0.data, nullptr);
-						info_->pic0.dataLoaded = true;
-						ReadFileToString(&umd, "/PSP_GAME/PIC1.PNG", &info_->pic1.data, nullptr);
-						info_->pic1.dataLoaded = true;
+						info_->pic0.dataLoaded = ReadFileToString(&umd, "/PSP_GAME/PIC0.PNG", &info_->pic0.data, nullptr);
+						info_->pic1.dataLoaded = ReadFileToString(&umd, "/PSP_GAME/PIC1.PNG", &info_->pic1.data, nullptr);
 					}
 					if (info_->wantFlags & GAMEINFO_WANTSND) {
 						ReadFileToString(&umd, "/PSP_GAME/SND0.AT3", &info_->sndFileData, nullptr);
-						info_->pic1.dataLoaded = true;
+						info_->sndDataLoaded = true;
```

Comment:
Should this also be `=ReadFileToString(...)`?  Although maybe some of these previous flags meant "tried to load", I don't think it'll keep trying to reload the file from the ISO either way.

-[Unknown]

Reply:
Yeah, probably should be.